web/models.py

```
@@ -2787,3 +2787,105 @@ class ScheduledPost(models.Model):
+        super().save(*args, **kwargs)
+
+    def __str__(self):
+        return f"{self.user.username}'s {self.title}"
+
+
+class LearningMapNode(models.Model):
+    NODE_TYPES = [
+        ("tracker", "Progress Tracker"),
+        ("course", "Course"),
+        ("milestone", "Custom Milestone"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)